src/components/staticcard/StaticCard.jsx

```diff
@@ -0,0 +1,79 @@
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import StaticCard1 from "../../assets/static_card_image1.png";
+import StaticCard2 from "../../assets/static_card_image2.png";
+import StaticCard3 from "../../assets/static_card_image3.png";
```

Comment:
StaticCard1, StaticCard2, etc doesn't make sense. They bring confusion between your `StaticCard` component and images.

Reply:
ok Sir